rules/unicorn.js

```diff
@@ -1,102 +1,49 @@
 export default {
   rules: {
     'unicorn/better-regex': 'error',
-    'unicorn/catch-error-name': 'error',
     'unicorn/consistent-destructuring': 'error',
-    'unicorn/consistent-function-scoping': 'error',
     'unicorn/custom-error-definition': 'error',
     // Code should be formatted using Prettier.
     'unicorn/empty-brace-spaces': 'off',
-    'unicorn/error-message': 'error',
-    'unicorn/escape-case': 'error',
     'unicorn/expiring-todo-comments': [
       'error',
       { allowWarningComments: false, ignoreDatesOnPullRequests: false },
     ],
-    'unicorn/explicit-length-check': ['error', { 'non-zero': 'greater-than' }],
     'unicorn/filename-case': ['error', { case: 'camelCase' }],
     // No good use cases yet.
     'unicorn/import-style': 'off',
-    'unicorn/new-for-builtins': 'error',
     // Covered by eslint-comments/no-unlimited-disable.
     'unicorn/no-abusive-eslint-disable': 'off',
     // Unnecessary with TypeScript.
     'unicorn/no-array-callback-reference': 'off',
-    'unicorn/no-array-for-each': 'error',
     // Covered by functional/no-this-expressions.
     'unicorn/no-array-method-this-argument': 'off',
-    'unicorn/no-array-push-push': 'error',
     'unicorn/no-array-reduce': 'off',
-    'unicorn/no-console-spaces': 'error',
-    'unicorn/no-for-loop': 'error',
-    'unicorn/no-hex-escape': 'error',
-    'unicorn/no-instanceof-builtins': 'error',
-    'unicorn/no-keyword-prefix': 'off',
-    'unicorn/no-lonely-if': 'error',
     // Ternary operators help write immutable code.
     'unicorn/no-nested-ternary': 'off',
-    'unicorn/no-new-array': 'error',
     // This is covered by @typescript-eslint/no-deprecated
     'unicorn/no-new-buffer': 'off',
     'unicorn/no-null': 'off',
     // Unnecessary with TypeScript.
     'unicorn/no-object-as-default-parameter': 'off',
-    'unicorn/no-process-exit': 'off',
```

Comment:
this looks useful to me, and there's not that many usages across our codebases, so I think it's fine if it's disabled on a case-by-case basis